popover-api/popover-hint/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Popover hint example</title>
+    <link href="index.css" rel="stylesheet" />
+    <script src="index.js" defer></script>
+  </head>
+  <body>
+    <div id="wrapper">
+      <section id="button-bar">
+        <button
+          popovertarget="submenu-1"
+          popovertargetaction="toggle"
+          id="menu-1">
+          Menu A
+        </button>
+
+        <button
+          popovertarget="submenu-2"
+          popovertargetaction="toggle"
+          id="menu-2">
+          Menu B
+        </button>
+
+        <button
+          popovertarget="submenu-3"
+          popovertargetaction="toggle"
+          id="menu-3">
+          Menu C
+        </button>
+      </section>
+    </div>
+
+    <div id="submenu-1" popover="auto">
+      <button>Option A</button><br /><button>Option B</button>
+    </div>
+    <div id="submenu-2" popover="auto">
+      <button>Option A</button><br /><button>Option B</button>
+    </div>
+    <div id="submenu-3" popover="auto">
+      <button>Option A</button><br /><button>Option B</button>
+    </div>
+
+    <div id="tooltip-1" class="tooltip" popover="hint">Tooltip A</div>
+    <div id="tooltip-2" class="tooltip" popover="hint">Tooltip B</div>
+    <div id="tooltip-3" class="tooltip" popover="hint">Tooltip C</div>
+  </body>
```

Comment:
Can we add something here like:

https://github.com/mdn/dom-examples/blob/8884ad4ba7043fa9d15b05d38b3e15836f81d4bd/popover-api/toggle-help-ui/index.html#L26

Saying what to look out for? Along the lines of 'mouseover the buttons to see effect of `popover="hint"`, clicking shows & hides the `popover="auto"` ones with a light dismiss of the `popover="hint"` ones'?

Reply:
I've added a help paragraph in my latest commit. I didn't want to try to say too much about the behavior as it is quite complex and hard to explain in a small space. Instead, I've added a general description of what is happening, and linked to the MDN explanation (which will be published soon) for more details.

Reply:
Yeah that's great, tnx 👍🏻